nmrfx-bom/pom.xml

```diff
@@ -15,6 +15,10 @@
     <properties>
         <nd4j.version>1.0.0-beta7</nd4j.version>
         <dl4j.version>1.0.0-beta7</dl4j.version>
+        <openblas.version>0.3.20-1.5.8-SNAPSHOT</openblas.version>
+        <arpack.version>3.8.0-1.5.7</arpack.version>
+        <javacpp.version>1.5.8-SNAPSHOT</javacpp.version>
```

Comment:
@brucejohnson just to double check: this was previously using 1.5.7, but had the native builds from snapshot transitively from openblas.